pyproject.toml

```diff
@@ -0,0 +1,129 @@
+license = {file = "LICENSE"}
+keywords = ["django", "authentication", "digid", "eherkenning", "eidas", "dutch", "nl", "netherlands"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Framework :: Django",
+    "Framework :: Django :: 4.2",
+    "Intended Audience :: Developers",
+    "Operating System :: Unix",
+    "Operating System :: MacOS",
+    "Operating System :: Microsoft :: Windows",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+requires-python = ">=3.10"
+dependencies = [
+    "cryptography>=40.0.0",
+    "django>=4.2.0",
+    "django-sessionprofile",
+    "django-simple-certmanager",
+    "django-solo",
+    "lxml>=4.7.1",
+    "furl",
+    "maykin-python3-saml>=1.16.0",
+]
+
+[project.urls]
+Homepage = "https://github.com/maykinmedia/django-digid-eherkenning"
+Documentation = "https://django-digid-eherkenning.readthedocs.io/en/latest/"
+"Bug Tracker" = "https://github.com/maykinmedia/django-digid-eherkenning/issues"
+"Source Code" = "https://github.com/maykinmedia/django-digid-eherkenning"
+Changelog = "https://github.com/maykinmedia/django-digid-eherkenning/blob/master/docs/CHANGELOG.rst"
+
+[project.optional-dependencies]
+oidc = [
+    "mozilla-django-oidc-db>=0.18.0",
+]
+tests = [
+    "django-test-migrations",
+    "pytest",
+    "pytest-django",
+    "pytest-mock",
+    "responses",
+    "freezegun",
+    "tox",
+    "isort",
+    "black",
+    "flake8",
+]
+coverage = [
+    "pytest-cov",
+]
+docs = [
+    "sphinx",
+    "sphinx-rtd-theme",
+    "psycopg2",
+]
+release = [
+    "bump-my-version",
+    "twine",
+]
+
+[tool.setuptools.packages.find]
+include = ["digid_eherkenning*"]
+namespaces = false
+
+[tool.isort]
+profile = "black"
+combine_as_imports = true
+known_django = "django"
+known_first_party="digid_eherkenning"
+sections=["FUTURE", "STDLIB", "DJANGO", "THIRDPARTY", "FIRSTPARTY", "LOCALFOLDER"]
+
+[tool.pytest.ini_options]
+testpaths = ["tests"]
+DJANGO_SETTINGS_MODULE = "tests.project.settings"
+markers = [
+    "callback: additional configuration for the callback fixture",
+    "mock_backend: class/claims to be returned by the mock backend fixture",
+]
+
+[tool.bumpversion]
+current_version = "0.15.0"
+files = [
+    {filename = "pyproject.toml"},
+    {filename = "README.rst"},
+    {filename = "docs/conf.py"},
+]
+
+[tool.coverage.run]
+branch = true
+source = [
+    "digid_eherkenning"
+]
+omit = [
+    # migrations run while django initializes the test db
+    "*/migrations/*",
+]
+
+[tool.coverage.report]
+skip_covered = true
+exclude_also = [
+    "if (typing\\.)?TYPE_CHECKING:",
+    "@(typing\\.)?overload",
+    "class .*\\(.*Protocol.*\\):",
+    "@(abc\\.)?abstractmethod",
+    "raise NotImplementedError",
+    "\\.\\.\\.",
+    "pass",
+]
+omit = [
+    "*/migrations/*",
+]
+
+[tool.coverage.html]
+directory = "cover"
```

Comment:
the default of `htmlcov` is fine and it's the directory in the `.gitignore` file, no need to change that

```suggestion
```